### PR TITLE
Update external documentation

### PR DESCRIPTION
**Pull request by lucy-kind**
Updates to data repo docs based on external import process

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Links:
 Once the entity and schema mapping have been finalized, you prepare the artifacts.  This includes:
 
 1. StatisticalVariable MCF nodes (if any) checked into [schema repo](https://github.com/datacommonsorg/schema/tree/main/stat_vars)
```
Comment (pradh) [inline docs/README.md:47]: "These nodes maybe written by hand ... "  (what you have below)

File: docs/README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Links:
...
-2. Template MCF and corresponding cleaned tabular files (typically CSV)
+2. Template MCF and corresponding cleaned tabular files (typically CSV). MCF and Template MCF nodes may be written by hand when there are only a handful in number. Otherwise, these nodes can be generated via scripts.
```
Comment (pradh) [inline docs/README.md:48]: "Like StatisticalVariable MCF, the Template MCF can also be hand-written or script-generated."

File: docs/life_of_a_dataset.md [hunk 3]
```diff
@@ -21,7 +21,11 @@ The statistical variable DCIDs should be human-readable, encapsulating the
 meaning of its triples.  The naming rules are summarized in [this
 doc](https://docs.google.com/document/d/1YtmQu_gOXENt4W2lmlK-8Mtnw_XB3kZlLBjSN-gT0oE/preview#bookmark=kix.bdksa8hbv3si).
 
-When the variables are finalized, they get checked into the [schema
+You'll need approval from the Data Commons team on any new variables (see the 
+"Defining location and schema mapping" section 
+[here](https://github.com/datacommonsorg/data/blob/master/docs/README.md)). 
+
+When the variables are finalized, they get checked into the [schema
```
Comment (pradh) [inline docs/life_of_a_dataset.md:28]: This doc doesn't talk about the doc at all.  Should we add a section before "## Define SVs", that reiterate the need to go through the doc review to finalize mapping / SVs?

File: docs/life_of_a_dataset.md [hunk 4]
```diff
@@ -74,6 +78,8 @@ In the PR, please also include the validation results (`report.json` and `summar
 
 If you introduced new statistical variables, please create a Pull Request for them in the [schema repo](https://github.com/datacommonsorg/schema).
 
+After your PR is reviewed and approved, the Data Commons team will work with you to manifest the data in our internal system.
```
Comment (pradh) [inline docs/life_of_a_dataset.md:81]: s/data/"artifacts you created"/?

Comment (pradh) [reply docs/life_of_a_dataset.md:81]: This is probably important enough to be a sub-section after sending PRs?

**Review by pradh — COMMENTED** [review]
Thanks!

File: docs/life_of_a_dataset.md (hunk 4 above)
Comment (jehangiramjad) [reply docs/life_of_a_dataset.md:81]: +1

File: docs/life_of_a_dataset.md (hunk 3 above)
Comment (lucy-kind) [reply docs/life_of_a_dataset.md:28]: Sg, to confirm you mean add a section to the README.md doc, right?

File: docs/life_of_a_dataset.md (hunk 4 above)
Comment (lucy-kind) [reply docs/life_of_a_dataset.md]: Created new sub-section, PTAL

File: docs/life_of_a_dataset.md [hunk 5]
```diff
@@ -74,6 +78,10 @@ In the PR, please also include the validation results (`report.json` and `summar
 
 If you introduced new statistical variables, please create a Pull Request for them in the [schema repo](https://github.com/datacommonsorg/schema).
 
+## Manifesting Artifacts
+
+After your PR is reviewed and approved, the Data Commons team will work with you to manifest the artifacts you've created in our internal system.
```
Comment (jehangiramjad) [inline docs/life_of_a_dataset.md:83]: nit: should we also state that only after the artifacts are manifested will they be added to the KG and be accessible?

**Review by jehangiramjad — APPROVED** [review]
Thanks! Approving this modulo other people's comments etc.

File: docs/life_of_a_dataset.md (hunk 3 above)
Comment (lucy-kind) [reply docs/life_of_a_dataset.md:36]: added a new section for Import Design Review. Once I'm done updating the schema-less guide, we can link to that here too. For now, this should be sufficient!

File: docs/life_of_a_dataset.md [hunk 6]
```diff
@@ -74,6 +78,10 @@ In the PR, please also include the validation results (`report.json` and `summar
 
 If you introduced new statistical variables, please create a Pull Request for them in the [schema repo](https://github.com/datacommonsorg/schema).
 
+## Manifesting Artifacts
```
[the remark this answers is not in the document]
Comment (lucy-kind) [reply docs/life_of_a_dataset.md:89]: added explanation

File: docs/life_of_a_dataset.md [hunk 7]
```diff
@@ -74,6 +86,10 @@ In the PR, please also include the validation results (`report.json` and `summar
...
+
+After your PR is reviewed and approved, the Data Commons team will work with you to manifest (e.g. generate for inclusion in our model) the artifacts you've created in our internal system. Only after the artifacts are manifested will they be added to the KG and be accessible?
```
Comment (beets) [inline docs/life_of_a_dataset.md:91]: Can we undo the trailing question mark?

We also don't use the term KG, rather Data Commons Graph (see docs.datacommons.org)

File: docs/representing_statistics.md [hunk 8]
```diff
@@ -40,8 +40,8 @@ gender: dcs:Female
 ```
 
 Before we get into the sematics of the above representation, as a quick primer
-to the MCF syntax: `Node` is a sentinel that defines a node in the KG, the LHS
-things (`typeOf`, `measuredProperty`, etc.) are DC Property entities, the RHS
+to the MCF syntax: `Node` is a sentinel that defines a node in the KG (Knowledge Graph), the left hand-side
```
Comment (beets) [inline docs/representing_statistics.md:43]: similarly can we update KG here?

**Review by beets — COMMENTED** [review]
thanks for updating the docs! some comments for consistency with our docs site.

**Comment by lucy-kind** [comment]
Thanks Carolyn! Updated PR here:
https://github.com/datacommonsorg/data/pull/797

All the best,

Lucy

**Pull request merged**